### PR TITLE
Changed "#Articles to list" to Use Global parameter

### PR DESCRIPTION
**Pull request by Shubhamverma2796**
Pull Request for Issue #36109.

### Summary of Changes

Changed #Articles to list from 5,10,....100 to Use Global parameter.

### Testing Instructions

First of all set the default configuration for # Article to list by heading over to:

#### Step 1:
1. System-->Global Configuration-->Articles-->List layouts.
2. Set any number 5,10,15,20...100, all for # Articles to list.
3. Save the configuration.
Test that you are able to save the configuration without any error.

#### Step 2:
1.Menus-->Article Category List--> List Layouts.
2. Verify that the # Articles to list is using the global parameter which you set in the first step.

### Actual result BEFORE applying this Pull Request

Screenshot (337)

No way to set the global parameters for # Articles to list.

Screenshot (334)

No way to use the "Use Global".

#### After step 1:
Screenshot (338)

Global configurations can be set for # Articles to list.

#### After step 2:

Screenshot (336)

"Use Global" can be used.

### CODE REVIEW

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 3fa1723e5681f5a542e1937a260ad7c610b137be<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36790">issues.joomla.org/tracker/joomla-cms/36790</a>.</sub>

File: administrator/components/com_content/config.xml [hunk 1]
```diff
@@ -890,6 +890,24 @@
...
+				<option value="5">J5</option>
+				<option value="10">J10</option>
+				<option value="15">J15</option>
+				<option value="20">J20</option>
+				<option value="25">J25</option>
+				<option value="30">J30</option>
+				<option value="50">J50</option>
+				<option value="100">J100</option>
+				<option value="0">JALL</option>
+			</field>
```
Comment (Quy) [inline administrator/components/com_content/config.xml]:
```suggestion
		<field
			name="display_num"
			type="list"
			label="JGLOBAL_NUMBER_ITEMS_LIST_LABEL"
			default="10"
			validate="options"
			>
			<option value="5">J5</option>
			<option value="10">J10</option>
			<option value="15">J15</option>
			<option value="20">J20</option>
			<option value="25">J25</option>
			<option value="30">J30</option>
			<option value="50">J50</option>
			<option value="100">J100</option>
			<option value="0">JALL</option>
		</field>
```

**Comment by pritam825** [comment]
I have tested this item :white_check_mark: successfully on a7a23ec5d548797369dea95c7ad20b1953b7f36b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36790">issues.joomla.org/tracker/joomla-cms/36790</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36790">issues.joomla.org/tracker/joomla-cms/36790</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**